### PR TITLE
Automatic slot removal docs

### PR DESCRIPTION
**Pull request by danieltprice**
Unused replication slots topic:
https://neon-next-git-dprice-replication-slot-warnings-neondatabase.vercel.app/docs/guides/logical-replication-neon#unused-replication-slots

Added the note seen here to other replication topics:
https://neon-next-git-dprice-replication-slot-warnings-neondatabase.vercel.app/docs/guides/logical-replication-airbyte#create-a-replication-slot

### CODE REVIEW

**Review by YanicNeon — APPROVED** [review]
Looks good to me. Approved.

File: content/docs/guides/logical-replication-neon.md [hunk 1]
```diff
@@ -337,14 +341,27 @@ If you require different values for these parameters, please contact Neon suppor
...
 ### Unused replication slots
 
-Neon removes _unused_ replication slots after a period of time to avoid unnecessary retention of write-ahead logs, which prevents removing data snapshots that are no longer required.
+To prevent storage bloat, **Neon automatically removes an _inactive_ replication slot if you have other _active_ replication slots**. Removal occurs after 75 minutes. 
+
+If you have only one replication slot, and that slot becomes inactive, it is not removed due to inactivity because a single replication slot does not create bloat. If you find that your single replication slot has been removed, please contact [Neon Support](/docs/introduction/support) for remediation.
+
+#### What causes a replication slot to become inactive? 
+
+An _inactive_ replication slot is often the result of a _dead subscriber_, where the replication slot is not dropped after a subscriber is deactivated or becomes unavailable. An inactive replication slot can also result from a replication delay configured on the subscriber. For example, some subscribers allow you to configure the frequency of replication or set a replication delay to minimize usage.
```
Comment (arssher) [inline content/docs/guides/logical-replication-neon.md:350]: Let's start this paragraph with ''An _inactive_ replication slot is the one which doesn't acknowledge progress on flush_lsn (as reported by same named field in pg_stat_replication)" for a long time.

File: content/docs/guides/logical-replication-neon.md [hunk 2]
```diff
@@ -337,14 +341,27 @@ If you require different values for these parameters, please contact Neon suppor
...
+
+#### How to avoid removal of inactive replication slots
+
+If you have more than one replication slot and want to avoid having "inactive" replication slots removed automatically, ensure that the replication frequency for your "additional" subscribers is less than 75 minutes.
```
Comment (arssher) [inline content/docs/guides/logical-replication-neon.md:354]: "replication frequency" sounds too fuzzy. How about "ensure that subscriber reports progress on flush_lsn regularly, and that replication connection doesn't disappear for more than 75m"? Also, let's add "If this period is not enough, please request support to increase it" here as well.

**Review by arssher — APPROVED** [review]

File: content/docs/guides/logical-replication-neon.md (hunk 1 above)
Comment (danieltprice) [reply content/docs/guides/logical-replication-neon.md:350]: Fixed.

**Comment by arssher** [comment]
Thanks, LGTM!

**Pull request merged**